### PR TITLE
DKTA Atlas Beta

### PR DESCRIPTION
**Pull request by kaikiwitz**

### CODE REVIEW

File: instances/atlas/brainAtlas/MarsAtlas.jsonld [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      ],
+      "custodian": [
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/person/brovelliAndrea"
+            }
+      ],
+      "description": "MarsAtlas is a model of cortical parcellation. It can be applied to any cortical surface via the HipHop parameterization pipeline, available in the BrainVisa Cortical Surface Toolbox, under the Anatomy category.",
+      "digitalIdentifier": [
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/digitalIdentifier/DOI_Mars_cortex_10.1002.hbm.23121"
```
Comment (lzehl) [inline instances/atlas/brainAtlas/MarsAtlas.jsonld]: DOIs do not have to be registered as separate entities in openMINDS. You can just directly use a DOI as "@id", e.g.:
"@id": "https://doi.org/10.1002/hbm.23121"

File: instances/atlas/brainAtlas/MarsAtlas.jsonld [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            }
+      ],
+      "fullName": "MarsAtlas",
+      "hasTerminology": {
+            "@type": "https://openminds.ebrains.eu/sands/ParcellationTerminology",
+            "definedIn": null,
+            "hasEntity": [
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/VCcm"
```
Comment (lzehl) [inline instances/atlas/brainAtlas/MarsAtlas.jsonld]: The entities "@id" should always start with the abbreviation of the atlas, e.g.: 
"@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Mars_VCcm"

File: instances/atlas/brainAtlas/MarsAtlas.jsonld [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/Mars_v2_cortexAndSubcortex"
+            }
+      ],
+      "homepage": "https://meca-brain.org/software/marsatlas/",
+      "shortName": "MarsAtlas",
+      "@context": {
+            "@vocab": "https://openminds.ebrains.eu/vocab/"
+      }
+}
```
Comment (lzehl) [inline instances/atlas/brainAtlas/MarsAtlas.jsonld]: JSONs need a newline at the end to be standard conform (unfortunately Python does not do this automatically; you have to explicitly add a line when saving a JSON)

File: instances/atlas/brainAtlas/MarsAtlas.jsonld [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/Mars_v1_cortex"
+            },
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/Mars_v2_cortexAndSubcortex"
+            }
+      ],
+      "homepage": "https://meca-brain.org/software/marsatlas/",
+      "shortName": "MarsAtlas",
+      "@context": {
```
Comment (lzehl) [inline instances/atlas/brainAtlas/MarsAtlas.jsonld]: should go to the top (just sort the keys before saving the JSON)

File: instances/atlas/brainAtlas/MarsAtlas.jsonld (hunk 2 above)
Comment (lzehl) [reply instances/atlas/brainAtlas/MarsAtlas.jsonld]: (I think the instances for the PEs actually have the correct "@id") only the here in the atlas it is wrong)

File: instances/atlas/brainAtlas/MarsAtlas.jsonld [hunk 5]
```diff
@@ -0,0 +1,190 @@
+{
```
Comment (lzehl) [inline instances/atlas/brainAtlas/MarsAtlas.jsonld:1]: please reformat to tab == 2 spaces

**Review by lzehl — CHANGES_REQUESTED** [review]
Overall the instances for the MarsAtlas look correct :) 
There are only some minor issues that need correction:

1) Please reformat the JSON to tab == 2 spaces.
2) Please sort the keys of the JSON before saving.
2) Please add a newline (empty line) at the end of each JSON (note: in Python this is not done automatically for some reason, you have to explicitly do this in your code).
3) Please do not create instances for DOIs in openMINDS but rather use reference the DOI by stating the DOI in the "@id" directly.
4) For ParcellationEntity and ParcellationEntityVersion please follow the naming convention for the "@id" (.../ATLASABBR(_ATLASVER)_REGIONNAME; in lower camelCase)

File: helper_code/DKTDataScrape.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import requests
```
Comment (lzehl) [inline helper_code/DKTDataScrape.py]: Please exclude this code from the PR

File: helper_code/DKT_data_structures.py [hunk 7]
```diff
@@ -0,0 +1,56 @@
+import MarsDataScrape
```
Comment (lzehl) [inline helper_code/DKT_data_structures.py]: Please exclude this code from the PR

File: helper_code/Mars_data_structures.py [hunk 8]
```diff
@@ -0,0 +1,40 @@
+import MarsDataScrape
```
Comment (lzehl) [inline helper_code/Mars_data_structures.py]: Please exclude this code from the PR

**Review by lzehl — CHANGES_REQUESTED** [review]
Please exlude all code that you used to generate the instances from this PR (it should only include the openMINDS instances, JSONLDs).

**Review by kaikiwitz — COMMENTED** [review]
added the requiered changes for the MarsAtlas and generated the DKT aplha version

File: instances/atlas/brainAtlas/MarsAtlas.jsonld (hunk 1 above)
Comment (kaikiwitz) [reply instances/atlas/brainAtlas/MarsAtlas.jsonld]: fixed

File: instances/atlas/brainAtlas/MarsAtlas.jsonld (hunk 3 above)
Comment (kaikiwitz) [reply instances/atlas/brainAtlas/MarsAtlas.jsonld]: fixed

File: instances/atlas/brainAtlas/MarsAtlas.jsonld (hunk 2 above)
Comment (kaikiwitz) [reply instances/atlas/brainAtlas/MarsAtlas.jsonld]: fixed

File: instances/atlas/brainAtlas/MarsAtlas.jsonld (hunk 4 above)
Comment (kaikiwitz) [reply instances/atlas/brainAtlas/MarsAtlas.jsonld]: fixed

File: instances/atlas/brainAtlas/MarsAtlas.jsonld (hunk 5 above)
Comment (kaikiwitz) [reply instances/atlas/brainAtlas/MarsAtlas.jsonld:1]: fixed

**Comment by kaikiwitz** [comment]
> Please exlude all code that you used to generate the instances from this PR (it should only include the openMINDS instances, JSONLDs).

**Review by lzehl — CHANGES_REQUESTED** [review]
+ keep lowerCamelCase for lookuplabels, filenames, and "@id"
+ introduce parent PEs for PEVs
+ merge hemispheres (left/right) and criteria type (deterministic/probabilistic) for the same brain region as annotations of the PEV (cf. [JBA_v3.0.2-Colin27_Area-hIP1](https://github.com/HumanBrainProject/openMINDS_SANDS/blob/v3/instances/atlas/parcellationEntityVersion/JBA_v3.0.2/JBA_v3.0.2-Colin27_Area-hIP1_PM-v7.2.jsonld))

DKTA versioning:
+ DKTA_b101-MNI152 (is alternative version of DKTA_b101-fsaverage)
+ DKTA_b101-fsaverage (is alternative version of DKTA_b101-MNI152, is new version of DKTA_b40-fsaverage)
+ DKTA_b40-fsaverage

**Comment by kaikiwitz** [comment]
I added new commits, the DKTA Atlas should now be updateded. 
 I will also manage the Mars Atlas in a seperate pull request to make it easier to review. 
Let me know what you think about the updates

> * keep lowerCamelCase for lookuplabels, filenames, and "@id"
> 
>     * introduce parent PEs for PEVs
> 
>     * merge hemispheres (left/right) and criteria type (deterministic/probabilistic) for the same brain region as annotations of the PEV (cf. [JBA_v3.0.2-Colin27_Area-hIP1](https://github.com/HumanBrainProject/openMINDS_SANDS/blob/v3/instances/atlas/parcellationEntityVersion/JBA_v3.0.2/JBA_v3.0.2-Colin27_Area-hIP1_PM-v7.2.jsonld))
> 
> 
> DKTA versioning:
> 
>     * DKTA_b101-MNI152 (is alternative version of DKTA_b101-fsaverage)
> 
>     * DKTA_b101-fsaverage (is alternative version of DKTA_b101-MNI152, is new version of DKTA_b40-fsaverage)
> 
>     * DKTA_b40-fsaverage

**Pull request closed without merging**